src/StoreManager/reducer.ts

```diff
@@ -31,12 +32,18 @@ export const reducer = function (
         case LOGIN:
             return produce(state, (draftState: State) => {
                 draftState.isLoogedIn = true
-                localStorage.setItem('user',JSON.stringify(state.forms.login))
-                draftState.user = state.forms.login
+                localStorage.setItem('user',JSON.stringify(action.payload.user))
+                if (action.payload.user) {
+                    draftState.user = action.payload.user
+                }
```

Comment:
Need to add guard before `JSON.stringify(action.payload.user)`

```suggestion
                const curUser = action?.payload?.user;
                if (curUser) {
                    localStorage.setItem('user',JSON.stringify(curUser));
                    draftState.user = curUser;
                }
```